### ISSUE TITLE
Thumbnail image displaying wrong image

### ISSUE THREAD

**Issue by carlycko** [cabrerahector/wordpress-popular-posts#86]
For some reason when displaying my popular posts the plugin is using the wrong thumbnail image. its set to use the featured image, says it is, but it is showing as the wrong one on my site.

**Comment by cabrerahector** [comment]
It's kinda difficult for me to tell what might be going on over there with
so little information from your part :P

Try clearing WPP's thumbnail cache (Settings > WordPress Popular Posts >
Tools) and see if it fixes the problem. If it doesn't, then please share
your site's URL at least so I can go take a look.

**Comment by carlycko** [comment]
Haha sorry. I did try clearing the cache. I changed the settings for now to display something else so its not the wrong posts showing live.

The issue seems bizarre. I see the image path in the source code - I downloaded that photo from the image path location off the site, opened it, its not the one thats displaying  http://prntscr.com/8lzt2s
I'm not sure why its displaying a totally different image, or where that image is coming from - if you have any ideas?
